inference-engine/src/transformations/src/transformations/convert_space_to_batch.cpp

```diff
@@ -120,6 +123,8 @@ void ngraph::pass::ConvertSpaceToBatch::convert_space_to_batch() {
 void ngraph::pass::ConvertSpaceToBatch::convert_space_to_batch_by_elements() {
     auto space_to_batch = ngraph::pattern::wrap_type<ngraph::opset3::SpaceToBatch>();
     ngraph::matcher_pass_callback callback = [this](pattern::Matcher& m) {
+        OV_ITT_SCOPED_TASK(ngraph::pass::itt::domains::nGraphPass_LT, "ngraph::pass::ConvertSpaceToBatch");
```

Comment:
this code has slightly different approach, lets call it 
```suggestion
        OV_ITT_SCOPED_TASK(ngraph::pass::itt::domains::nGraphPass_LT, "ngraph::pass::ConvertSpaceToBatchByElements");
```

Reply:
fixed